packages/story-editor/src/utils/getInsertedElementSize.ts

```diff
@@ -24,28 +24,31 @@ import {
   DEFAULT_DPR,
 } from '@googleforcreators/units';
 import { getDefinitionForType } from '@googleforcreators/elements';
+import type {
+  Element,
+  ElementType,
+  Direction,
+} from '@googleforcreators/elements';
+import type { Resource } from '@googleforcreators/media';
 
-const RESIZE_WIDTH_DIRECTION = [1, 0];
+const RESIZE_WIDTH_DIRECTION: Direction = [1, 0];
 
 /**
- * @param {?number|undefined} value The value.
- * @return {boolean} Whether the value has been set.
+ * @param The value.
+ * @return  Whether the value has been set.
  */
-function isNum(value) {
+function isNum(value: unknown) {
```

Comment:
The return type here could even be a type guard:

```suggestion
function isNum(value: unknown): value is number {
```

Not sure if it makes a difference though.

Reply:
Not sure what value this would add here...

Reply:
[Type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) like this are useful for type narrowing